src/libraries/System.Private.CoreLib/src/System/Environment.Windows.cs

```diff
@@ -358,5 +360,43 @@ private static unsafe string[] SegmentCommandLine(char* cmdLine)
 
             return arrayBuilder.ToArray();
         }
+
+        /// <summary>
+        /// Get the CPU usage, including the process time spent running the application code, the process time spent running the operating system code,
+        /// and the total time spent running both the application and operating system code.
+        /// </summary>
+        [SupportedOSPlatform("maccatalyst")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        public static ProcessCpuUsage CpuUsage
+        {
+            get
+            {
+                using (SafeProcessHandle handle = GetProcessHandle())
+                {
+                    Debug.Assert(!handle.IsInvalid);
+
+                    if (!Interop.Kernel32.GetProcessTimes(handle.DangerousGetHandle(), out long create, out long exit, out long kernel, out long user))
```

Comment:
This should use `GetProcessTimes` overload that that takes IntPtr as the first argument. Notice that we do that e.g. in RuntimeEventSourceHelper already.

Creating SafeHandle to get the utilization of the current process is unnecessary overhead.

Reply:
Done. This nicely simplified the implementation. Thanks!